corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -146,8 +170,16 @@ def test_submission_error_log_migration(self):
         self._compare_diffs([])
 
     def test_hqsubmission_migration(self):
```

Comment:
Based on the git history, docs with type 'HQSubmission'
are just normal forms that were submitted by HQ ie. system forms
and were mostly used for:

* force closing cases: 37398d6
* submitting stock reports: 4f609c9, 23e729c